doc/index.rst

```diff
@@ -54,14 +46,16 @@ is slightly modified:
 
     python setup.py install --user
```

Comment:
I definitely agree that this has its pitfalls. Here's a bit of the backstory on this: we plan to run LR using the stable version of `acis_thermal_check` and that of the various model checker tools, but from installations which we manage as the ACIS ops team instead of the ones which are installed in flight Ska, though of course we would still be using that Python stack. 

We're still working on the details, but the current idea was to install the packages with `--user` into a particular directory under `/data/acis/LoadReviews`, but when LR is run or we run thermal model tests we run in an environment which has `PYTHONUSERBASE` set to that location. We would not (under `acisdude` or any of our individual accounts) have that env variable set otherwise to avoid messing around with those packages accidentally or using them when we didn't intend to.